irispy/sji_test.py

```diff
@@ -0,0 +1,36 @@
+import sji
+import matplotlib.pyplot as plt
+import numpy.ma as ma
+
+iris_dir = '/Users/shelbe/Documents/IRIS/data/AR12641/'
+fname = [iris_dir + 'iris_l2_20170228_053417_3600106076_SJI_1330_t000.fits',
```

Comment:
these should obviously not be hard coded. Has there been any discussion about how to handle test data?